extensions/prototype-content-with-issues-table/app.R

```diff
@@ -27,39 +29,51 @@ as_content_list <- function(content_df, client) {
   })
 }
 
-# checks to see if a content item has failed jobs within the last 30d, grabs 
-# usage data if it does, then compiles content, job, and usage data together 
-# into a tibble, returning it.
-get_failed_job_data <- function(item, usage) {
-  failed_jobs <- get_jobs(item) |>
-        # filter out successful and running jobs
-        filter(exit_code != 0 & status != 0 & !(is.na(end_time)))
-  
+# filters content jobs down to failures and sets content_recovered depending on 
+# whether or not the latest job ended in error
+filter_jobs <- function(jobs) {
```

Comment:
Minor, but could we give this a more descriptive name? `filter` could be filtering on a bunch of different dimensions.

Reply:
We talked about this in our 1:1 but to document what we talked about: something like `filter_failed_jobs()` would be evocative. The comment about this function signature is great, but the function name is important for communicating later on when we actually call the function. Yes, you can look up the function and find this, but that's one extra step when we are reading the code.